index.js

```diff
@@ -45,8 +45,13 @@ function getPackages(basePath, paths = ['@lblod/ember-rdfa-editor*']) {
 
 module.exports = {
   name: require('./package').name,
-  included: function (/* app */) {
+  included: function (app) {
     this._super.included.apply(this, arguments);
+
+    app.options.sassOptions = app.options.sassOptions || {};
+    app.options.sassOptions.includePaths =
+      app.options.sassOptions.includePaths || [];
```

Comment:
I don't think this part is needed, as it is already included in the `index.js` of `@appuniversum/ember-appuniversum`.
As for the the `includePaths` itself, I think it should be configured in the host/dummy app, not in `index.js`, like mentioned in https://appuniversum.github.io/ember-appuniversum/?path=/story/outline-getting-started--page. So in this case, in `ember-cli-build.js`.
It is weird though that using `["node_modules/@appuniversum/ember-appuniversum"]` as the value for `includePaths` does not seem to work correctly.

Reply:
Hmm, I didn't realise that was a thing. I'll try that as it indeed makes more sense.